mrgsolve.Rproj

```diff
@@ -1,4 +1,5 @@
 Version: 1.0
+ProjectId: 0063b92e-a050-4fd7-986e-be62f57eb1b1
```

Comment:
Just a heads up that this will conflict with the project ID from gh-1266.

Reply:
Thanks, @kyleam